### PR TITLE
fixes dumb things from trekchem removal pr and nerfs trophazole/rhigoxane

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

gives ambrosia trophazole and rhigoxane instead of the injector chems
gives stuffed legion omnizine since tricord was lost
ascepius snake now gives antihol because thats like the only heal toxins chem i could think of
nerfs trophazole and gives a small nerf to rhigoxane

## Why It's Good For The Game

ascepius snake should heal all
stuffed legion became just nutriment capsaicin and vitamins, uncool
plants having injection chems when you are supposed to eat them is dumb
rhigoxane and trophazole are basically c3 chems, should be more of a c2

## Changelog
:cl:
fix: fixes problems from trekchems being removed
balance: nerfs rhigoxane and trophazole
/:cl:

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
man, that was quick. good work!

**Comment by Dawson1917** [comment]
New drinking challenge is to try getting ratfaced in the bar faster than the rod of ascepius can sober you

**Comment by ExcessiveUseOfCobblestone** [comment]
> fixes plants having injection chems because they dont work on ingestion, who the fuck grinds plants and injects themselves with them what the fuck, replaced with sal acid and oxalondrone

People who don't want the downsides associated with not using the proper method I'd imagine.

I don't think this is a good reason to replace them with C3s, either make your own C2 / Edit current C2s / Leave it in.

**Comment by Fikou** [comment]
> I don't really see an issue with this since you're still able to mass produce the reagent and botany does in fact have syringes.
i just dont like the idea that things you consume orally you are supposed to grind and inject to your body like a madman and chems that are only for injections are in the things you consume orally

**Comment by ExcessiveUseOfCobblestone** [comment]
you can still eat the plant though? You just won't get the healing properties.

It doesn't look like anything /bad/ happens when you don't inject them, they just don't work.

**Comment by Krysonism** [comment]
I do agree these chems need to be altered to fit the new system.

But i think these changes to trophazole and rhigox are ill advised.

This PR fails in making rhigoxane a type 2. It just makes it a shitty type 3 that will go from seeing marginal use to no use untill ointment, oxal and sulfa are nerfed.

I do think rhigoxane should eventually end up in category 2 since it is already a type of emergency chem.

As for trophazole it makes no sense thematically in cat 2, you wouldn't force-feed cake to someone who is mortally wounded. I'd like to see it's recipe be made harder and more emphasis put on the food part. I also dislike the lipolicide like effect for the OD, it is overused.

I don't think we need to jump the gun here. 

Focusing on the meds people actually use (brute pack/ointment, oxal/sali, styptic/sulfa) will make it easier to determine how these marginal chems should fit in to the greater system.

**Comment by SgtxRootbeer** [comment]
For type 2's, we can't make the recipie harder. It has to be trekchem-easy to make, otherwise it's not an emergency chem that can be pumped out en masse when needed, but rather so difficult that chemists ignore it.

**Comment by Krysonism** [comment]
Yes, that is why I am advocating making trophazole a proper type 3. 

I don't think any recipes need to be trekchem easy(except type 1 chems designed for ghetto crafting). 

Making hundreds of units of even a proper type 3; salbutamol is not hard or time consuming at all.

I've come to the conclusion that this notion of medbay being too dependant on chemists that can't keep up is largely a myth. I don't think I've ever ran out of meds in my numerous games as CMO.

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -739,7 +743,7 @@
 	description = "These restorative peptides not only speed up wound healing, but are nutrious as well!"
 	nutriment_factor = 10 * REAGENTS_METABOLISM // 33% less than nutriment to reduce weight gain
 	brute_heal = 3
-	burn_heal = 1
+	tox_heal = 1
```
Comment (CPTANT) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: Why

Comment (Fikou) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: having trophazole gives you toxin, this heals the toxin

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 2]
```diff
@@ -1259,27 +1259,32 @@
...
 	. = 1
 
 /datum/reagent/medicine/trophazole/on_transfer(atom/A, method=INGEST, trans_volume)
 	if(method != INGEST || !iscarbon(A))
+		M.adjustFireLoss(0.5*reac_volume)
 		return
 
 	A.reagents.remove_reagent(/datum/reagent/medicine/trophazole, trans_volume * 0.05)
 	A.reagents.add_reagent(/datum/reagent/medicine/metafactor, trans_volume * 0.25)
+	A.reagents.add_reagent(/datum/reagent/medicine/metafactor, trans_volume * 0.2)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: (x * 0.25) + (x * 0.2) = (x * 0.45)

**Comment by ExcessiveUseOfCobblestone** [comment]
may need to look at a different approach to cat 3s then that involves reagents not available in a dispenser like omni or mining reqs.

**Comment by Dawson1917** [comment]
Making trophazole give you toxin damage (yet ANOTHER source) if you don't eat is just awful. Running around balancing two to three different chems just because you accidentally shocked yourself or stepped on a shard or had a rough up in the bar or god knows what other minor thing is **not fun**. Being REQUIRED to eat food (there isn't food just sitting around medbay, so gg if someone else is healing you with trophazole) when taking troph is a layer of obnoxious effort that isn't required for every chem, let alone one that just deals with something as simple as brute.

**Comment by Krysonism** [comment]
> 
> 
> may need to look at a different approach to cat 3s then that involves reagents not available in a dispenser like omni or mining reqs.
> 
> I forgot about the "feature" of replacing batteries to 0-100 your energy supply for dispensers so gating behind that seems fairly irrelevant now.

I agree strongly agree, the chemists get inducers every round so energy is a non issue for them.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: wait i didnt want to put that twice

**Comment by Krysonism** [comment]
Is there really a need to butcher these very niche chems when cobby is already reworking all the healing chems?

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm reworking all of Category 2's (coming soon to medbays near you), if that PR doesn't pan out we can look at this again.

**Comment by ExcessiveUseOfCobblestone** [comment]
Well all of the meds, Cat 2's are the first.

**Pull request closed without merging**